internal/controller/controllers/clusterdeployments_controller.go

```diff
@@ -583,6 +582,12 @@ func (r *ClusterDeploymentsReconciler) updateStatus(ctx context.Context, cluster
 	} else {
 		setClusterConditionsUnknown(cluster)
 	}
+
+	if cluster.Spec.Installed {
+		cluster.Status.APIURL = fmt.Sprintf("https://api.%s.%s:6443", cluster.Spec.ClusterName, cluster.Spec.BaseDomain)
+		cluster.Status.WebConsoleURL = fmt.Sprintf("https://console-openshift-console.apps.%s.%s", cluster.Spec.ClusterName, cluster.Spec.BaseDomain)
```

Comment:
Could you make it a common function for other usages on 
https://github.com/openshift/assisted-service/blob/a1e013fc51d0f6272c1b0ae386c0b1a67f701696/internal/cluster/transition.go#L359
https://github.com/openshift/assisted-service/blob/f9dd444e20529ea1d642b1698b9ca4c19a25b912/internal/bminventory/inventory.go#L3448

Reply:
Good idea

Reply:
Done